plugins/dev-tools/src/playground/index.js

```diff
@@ -242,29 +246,47 @@ export function createPlayground(
 
         // Register default tabs.
         const tabs = {
+          'JSON': registerGenerator(
+              'JSON',
+              'json',
+              (ws) => {
```

Comment:
Will clang-format preserve this more compact formatting?:
```suggestion
          'JSON': registerGenerator( 'JSON', 'json', (ws) => {
```
(Here and below.)

Reply:
I think it will, but doesn't that break the [recommendation](https://google.github.io/styleguide/jsguide.html#formatting-where-to-break) of wrapping at a higher syntactic level? I know this takes up more space, but personally I find it more readable.